data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -353,7 +354,7 @@
     field_names.ADJACENT_TRACT_SCORE_ABOVE_DONUT_THRESHOLD: "ADJ_ET",
     field_names.TRACT_PERCENT_NON_NATURAL_FIELD_NAME
     + field_names.PERCENTILE_FIELD_SUFFIX: "IS_PFS",
-    field_names.NON_NATURAL_LOW_INCOME_FIELD_NAME: "IS_ET",
+    field_names.NON_NATURAL_PCTILE_THRESHOLD: "IS_ET",
```

Comment:
@lucasmbrown-usds 

The frontend is still showing that this is false when the value is above 90.

![Screen Shot 2023-01-12 at 11 16 00 AM](https://user-images.githubusercontent.com/86254807/212160528-666e02eb-fd5f-462a-a442-4fa0b316a7e7.png)